.github/workflows/haskell.yml

```diff
@@ -32,7 +32,7 @@ jobs:
 
     env:
       # Modify this value to "invalidate" the cabal cache.
-      CABAL_CACHE_VERSION: "2024-07-26"
+      CABAL_CACHE_VERSION: "2024-09-30"
```

Comment:
Why this was needed?

Reply:
It wasn't I thought it was when I was fighting the CI